source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,120 @@
+---
+title: Reporting 
+weight: 70
+---
+
+# Reporting 
+
+You can extract reporting information from GOV.UK Pay manually, or via the
+API.
+
+## Manual reporting
+
+You can sign in to your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) and export
+transaction information in a CSV file. 
+
+For a given account, select __Transactions__ in the __Dashboard__ page.
+
+You can filter transactions using the available search criteria, and then
+select __Download all transaction details as a CSV file__. 
+
+You can process CSV files using spreadsheet software.
+
+## Reporting via the GOV.UK Pay API 
+
+Before reading this section, you should read the section describing [Making
+payments](/making_payments). 
+
+You can develop software to get data from the GOV.UK Pay API and import it into 
+a finance system. This could remove routine manual operations from your workflow and
+lead to significant cost savings for your organisation. 
+
+You can use the GOV.UK Pay API to:
+
+* get information about a single payment
+* generate a list of payments matching search criteria  
+
+## Get information about a single payment
+
+`GET /v1/payments/paymentId`
```

Comment:
This is fine — not sure if we have a convention for indicating that `paymentId` is a placeholder

Reply:
We should start one: I'd like to look at that separately across the whole document soon after we eventually merge this PR.

Reply:
@bahmady I'd also be interested in what you decide for this 👍